tf_encrypted/keras/optimizers.py

```diff
@@ -0,0 +1,24 @@
+"""TFE Keras optimizers"""
+from tensorflow.keras import backend as K
+
+import tf_encrypted as tfe
+
+class SGD():
+  def __init__(self, lr=0.01):
+    self.lr = lr
+
+  def apply_gradients(self, var, grad):
+    sess = K.get_session()
```

Comment:
wouldn't this give you a `tf.Session` instead of `tfe.Session`?

Reply:
Fixed - good catch